presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -3207,4 +3213,9 @@ public static CreateView.Security getDefaultViewSecurityMode(Session session)
     {
         return session.getSystemProperty(DEFAULT_VIEW_SECURITY_MODE, CreateView.Security.class);
     }
+
+    public static boolean isJoinPrefilterEnabled(Session session)
```

Comment:
I think this must be added in Featuresconfig as a config property

Reply:
No - this is not a feature of the cluster/deployment. This is a query level optimization so I don't want to add it there

Reply:
I see anyways its false now, but in future if we make this cost based, it might become a cluster level feature.

Reply:
I'm generally against these feature config stuff for query level optimizations because HBO can be controlled at query level

Reply:
Separately we need HBO configuration that can be used to control what features it supports. @feilong-liu  and I were talking about it